oop/homework.py

```diff
@@ -39,31 +42,63 @@ class Cat:
     """
 
     def __init__(self, age):
-        pass
+        self.age = age
+        self.average_speed = self._set_average_speed()
+        self.saturation_level = 50
 
     def eat(self, product):
-        pass
+        if product == 'fodder':
```

Comment:
what if U create dict with matching params `my_dict = {'apple': 5, .....}` 
and simply do that: `self_increase_satuation_level(my_dict.get(product, 0))`

Reply:
Great idea! Thanks!